code/modules/projectiles/updated_projectiles/guns/energy.dm

```diff
@@ -241,4 +240,4 @@
 	if(!isYautja(user))
 		to_chat(user, "<span class='warning'>You have no idea how this thing works!</span>")
 		return
-	return ..()
\ No newline at end of file
+	return ..()
```

Comment:
scream emoji

Reply:
Something wrong? :scream:

Reply:
the more I think, the more I'm confused on which one gives the early return.